### PR TITLE
tide: Add more triggers

### PR DESCRIPTION
**Pull request by NaveenKarippai**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`tides: added more triggers`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{SpiceRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

The tide spice should be triggered for more queries as suggested
on issue tracker. It could be done by adding more trigger words.

Provide an overview of the changes this pull request introduces.

* added more triggers

###### Which issues (if any) does this fix?

See also: #2737

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@tagawa)

---

Instant Answer Page: https://duck.co/ia/view/tides

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mattr555

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Spice/Tides.pm [hunk 1]
```diff
@@ -8,6 +8,7 @@ spice is_cached => 1; #we can cache the response and just filter out the tide ev
 spice to => 'http://api.wunderground.com/api/{{ENV{DDG_SPICE_WUNDERGROUND_APIKEY}}}/tide/q/$1.json?callback={{callback}}';
 
 triggers any => "tide", "tides";
+triggers start => "tides for", "tide in", "tide times in";
```
Comment (gaulrobe) [inline lib/DDG/Spice/Tides.pm]: Do we want a `tide for` and a `tides in` here too?

Comment (NaveenKarippai) [inline lib/DDG/Spice/Tides.pm]: done!

File: lib/DDG/Spice/Tides.pm [hunk 2]
```diff
@@ -8,6 +8,7 @@ spice is_cached => 1; #we can cache the response and just filter out the tide ev
 spice to => 'http://api.wunderground.com/api/{{ENV{DDG_SPICE_WUNDERGROUND_APIKEY}}}/tide/q/$1.json?callback={{callback}}';
 
 triggers any => "tide", "tides";
+triggers start => "tides for", "tide for", "tide in", "tides in", "tide times in";
```
Comment (gaulrobe) [inline lib/DDG/Spice/Tides.pm]: Could you add some test cases for `tide for ` and `tides in` too? Thanks :smile:

Comment (NaveenKarippai) [inline lib/DDG/Spice/Tides.pm]: sure, the new commit object should add necessary changes. :8ball:

**Comment by moollaza** [comment]
@NaveenKarippai Thanks! This LGTM :+1:

@mattr555 any objects? If not, feel free to merge!

**Comment by mattr555** [comment]
:+1: @NaveenKarippai Thanks a lot for the help! :smile:

**Pull request merged**